### PR TITLE
Add dmx adapter

### LINKED ISSUE #682: DMX512 adapter
Hello,
It will be nice to see [DMX](https://en.wikipedia.org/wiki/DMX512) support.

Thanks

### PR DESCRIPTION
**Pull request by tim-hellhake**
resolves #682

### CODE REVIEW

**Comment by tim-hellhake** [comment]
> manifest.json is missing for "dmx-adapter"

I can prove it action!  :triumph:

```
> curl -L --silent https://github.com/tim-hellhake/dmx-adapter/releases/download/0.1.0/dmx-adapter-0.1.0-linux-x64.tgz | tar -tz
manifest.json
LICENSE
README.md
dmx-adapter
SHA256SUMS
```

**Comment by tim-hellhake** [comment]
Got it, the action wants a package folder in the archive.

File: addons/dmx-adapter.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  "description": "Connect your DMX devices",
+  "author": "Tim Hellhake",
+  "homepage_url": "https://github.com/tim-hellhake/dmx-adapter#readme",
+  "license_url": "https://raw.githubusercontent.com/tim-hellhake/dmx-adapter/master/LICENSE",
+  "primary_type": "adapter",
+  "packages": [
+    {
+      "architecture": "linux-arm",
+      "language": {
+        "name": "nodejs",
```
Comment (mrstegeman) [inline addons/dmx-adapter.json:13]: This should be `binary`, not `nodejs`.

**Review by mrstegeman — CHANGES_REQUESTED** [review]

File: addons/dmx-adapter.json (hunk 1 above)
Comment (tim-hellhake) [reply addons/dmx-adapter.json]: I wasn't if the addon proxy knows `binary`.

**Review by mrstegeman — APPROVED** [review]

File: addons/dmx-adapter.json (hunk 1 above)
Comment (mrstegeman) [reply addons/dmx-adapter.json]: I wasn’t either but it looks like it works. 😆

**Pull request merged**